pkg/constants/constants.go

```diff
@@ -24,6 +24,11 @@ const (
 	// TODO(#23): Use the kubernetes.io domain when graduating APIs to beta.
 	QueueAnnotation = "kueue.x-k8s.io/queue-name"
 
+	// ParentWorkloadAnnotation is the annotation used to mark child jobs to be
+	// controller by Kueue and ignored from scheduling. It's value is the name
+	// of the workload controller by the main job.
```

Comment:
is the annotation used to mark a Job as a child of a Workload. This can be used when a custom job CRD is composed of one or more kubernetes Jobs. When set, Kueue ignores this Job from admission. The value is the name of the Workload that controls this Job, in the same namespace.

Reply:
However, it's not totally ignored. If the workload doesn't exist or it's not admitted, we should suspend the job.

Reply:
rephrased, PTAL